hoomd/hpmc/integrate.py

```diff
@@ -66,6 +66,16 @@ class HPMCIntegrator(BaseIntegrator):
     TODO: Describe implicit depletants algorithm. No need to write this now,
     as Jens is rewriting the implementation.
 
+    .. rubric:: Writing type_shapes to GSD files.
+
+    Use a Logger in combination with a HPMC integrator and a GSD writer to write
+    ``type_shapes`` to the GSD file for use with OVITO::
+
+        mc = hoomd.hpmc.integrate.<Shape>(...)
+        log = hoomd.logging.Logger()
+        log.add(mc, quantities=['type_shapes'])
+        gsd = hoomd.write.GSD(..., log=log)
```

Comment:
Sometimes I think it's easier to understand "runnable" code and guess the general case from a specific example.
```suggestion
    ``type_shapes`` to the GSD file for use with OVITO. For example::

        mc = hoomd.hpmc.integrate.Sphere(seed=123)
        log = hoomd.logging.Logger()
        log.add(mc, quantities=['type_shapes'])
        gsd = hoomd.write.GSD(
            'trajectory.gsd', hoomd.trigger.Periodic(1000), log=log)
```